support/util/imagemetadata_test.go

```diff
@@ -166,7 +167,7 @@ func TestGetDigest(t *testing.T) {
 	}{
 		{
 			name:        "if failed to parse image reference",
-			imageRef:    "invalid-image-ref",
+			imageRef:    "::invalid-image-ref",
```

Comment:
Is this in purpose?

Reply:
Yes, because I changed how we obtain the image reference to use the ref.String() function, a name by itself is considered valid (before with the fmt.Sprintf() it was getting converted to something like: "//invalid-image-ref"). To still have an invalid ref, I added the extra "::"

Reply:
Gotcha, thanks